include/boiler/script-call-fr.md

```diff
@@ -1,106 +1,109 @@
-    Bonjour ici Céline Provins,
+Bonjour ici Céline Provins,
```

Comment:
Please check how this is rendered locally with `mkdocs serve`. If you do so, you'll see that the indentation of every line here is necessary.

Reply:
Can you please provide more details? I have checked using mkdocs serve, but I cannot identify the problem.